buildSrc/subprojects/performance/src/main/groovy/org/gradle/testing/DistributedPerformanceTest.groovy

```diff
@@ -54,10 +55,6 @@ import org.apache.commons.io.input.CloseShieldInputStream
 @CompileStatic
 @CacheableTask
 class DistributedPerformanceTest extends PerformanceTest {
-
```

Comment:
This is removed since we can fetch build id from TC env variable `BUILD_ID`. Since `PerformanceTest` needs it, too, the `buildId` property is moved to the superclass `PerformanceTest`.